src/main/java/com/artipie/asto/fs/FileStorage.java

```diff
@@ -131,4 +148,14 @@ public FileStorage(final Path path) {
     public CompletableFuture<Transaction> transaction(final List<Key> keys) {
         return CompletableFuture.completedFuture(new FileSystemTransaction(this));
     }
+
+    /**
+     * Resolves key to file system path.
+     *
+     * @param key Key to be resolved to path.
+     * @return Path created from key.
+     */
+    private Path path(final Key key) {
+        return Paths.get(this.dir.toString(), key.string());
+    }
```

Comment:
@olegmoz This method doesn't really belong in this PR. For future reference, please only implement the topic(s) required by the Issue. In this PR you were supposed to implement ``Storage.move()`` and test it, nothing else :)

Reply:
@amihaiemil had to use code duplicated multiple times in this class already, decided to do the right thing! Do not blame me for this :)